chapter_computational-performance/hybridize_vn.md

```diff
@@ -382,15 +383,15 @@ Things are slightly more tricky when it comes to models that resemble code more
 Basically hybridization needs to deal with control flow and Python overhead in a much more immediate manner. Moreover,
 -->
 
-Mọi thứ phức tạp hơn một chút khi nói đến các mô hình gần với mã nguồn.
-Về cơ bản việc hybrid hoá cần làm việc trực tiếp với luồng điều khiển và các chi phí tính toán của Python.
+Mọi thứ trở nên phức tạp hơn một chút khi làm việc với các mô hình gần với mã nguồn.
+Về cơ bản, việc hybrid hoá cần giải quyết trực tiếp luồng điều khiển và các chi phí tính toán của Python.
```

Comment:
Câu này mình chưa hiểu lắm

Reply:
à sorry mình định tránh lặp từ thôi, how bout this
```suggestion
Về cơ bản, việc hybrid hoá cần phải xem xét luồng điều khiển và các chi phí tính toán của Python.
```

Reply:
ok nhé